README.md

```diff
@@ -1,30 +1,48 @@
 # Kyma Community
 
-Welcome to the Kyma community.
-
 ## Overview
 
-This is an open repository for all Kyma contributors.
+Welcome to the Kyma community. This repository is your starting point for contributing code and content to Kyma.
+
+Read the document to learn:
+- which rules govern the Kyma community
+- how to submit contributions that follow the recommended flow, approved templates, and standards
+- who to contact for questions or suggestions
+
+## Prerequisites
+
+Before you learn how to contribute, read the [Code of Conduct](CODE_OF_CONDUCT.md) for rules of behavior for all contributors and maintainers in the community. Also see the Kyma [governance](governance.md) model to better understand the role of a maintainer and the process of becoming one.
+
+If you are interested in the Kyma team's main focus and its approach to product management, architecture, or the operational responsibility, read the Kyma [Manifesto](https://kyma-project.github.io/community/).
 
 ## Usage
 
-The purpose of this repository is to:
+Follow the `community` repository guidelines and templates to ensure that you contribute high-quality code and content to your project.
```

Comment:
My suggestion is to link to the guidelines and templates in the community repo. IMHO, it would be more likely for the contributor to click the link than to search for the guidelines.

Reply:
Sure I can, I just did not want to add to many links.